2.6/examples/replica/mongodb-clustered.json

```diff
@@ -0,0 +1,167 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+
```

Comment:
This would be identical to `2.4/examples/replica/mongodb-clustered.json`, if it wasn't for this lonely empty line...